projects/components/src/table/cells/expanded-detail/table-expanded-detail-row-cell-renderer.component.scss

```diff
@@ -2,7 +2,7 @@
 @import 'font';
 
 .table-expanded-cell-renderer {
-  padding: 0 0 16px 18px; // Using 18px for the right padding aligns the dotted line with the expander arrow
```

Comment:
does it still look nice?

Reply:
imo yes, you can check the video sent in slack